xbmc/platform/android/network/NetworkAndroid.h

```diff
@@ -72,4 +69,10 @@ class CNetworkAndroid : public CNetworkBase
   std::vector<CNetworkInterface*> m_interfaces;
   std::vector<CNetworkInterface*> m_oldInterfaces;
   CCriticalSection m_refreshMutex;
+
+  CNetworkInterface* m_defaultInterface = nullptr;
```

Comment:
is it possible to make this a unique_ptr or is it too involved in other parts of the code?

Reply:
I've tried, but this class is common to all platforms and copy operations are done in other parts of the code... a lot of changes, I think it's better to leave it as it is

Reply:
Yeah that's what I anticipated..thanks for checking anyway 👍